requirements/edx/base.txt

```diff
@@ -90,7 +90,7 @@ python-dateutil==2.1
 # We can also remove the fix to auth_url in third_party_auth/saml.py when that fix is included upstream.
 python-social-auth==0.2.12
 
-pytz==2015.2
+pytz>=2016.7
```

Comment:
w/r/t my tête–à–tête with @feanil , pinning makes a future database upgrade easier to find/diagnose, so I'll change the '>=' to a '==' for the sake of future generations.